process_pub_database/get_data_cpdl.py

```diff
@@ -219,9 +203,11 @@ def main():
 
 		except Exception as e: 
```

Comment:
A broad exception here is a bad idea - it's unclear why you're trying to catch it - network issues? fields not existing in the html body? It'd be better to have a specific try/catch around the specific thing that might fail, and have the exception that it catches be more specific.